sdk/core/perf/src/program.cpp

```diff
@@ -320,6 +320,13 @@ void Azure::Perf::Program::Run(
       // Take the corresponding proxy from the list in round robin
       parallelTest[i]->SetTestProxy(options.TestProxies[i % options.TestProxies.size()]);
     }
+    if (options.Insecure)
+    {
+#if defined(BUILD_TRANSPORT_WINHTTP_ADAPTER)
```

Comment:
Just printing an error seems wrong - why not fail at this point?

Reply:
I want the test to fail while validating the certificate.  This will just give a quick help about the error.
The performance tests are usually run with `Release` mode, so I want to print the error in console.
And I don't want to hide the error (by throwing before it happens).  The error should be seen and noticed :)

Reply:
The best way of guaranteeing that an error is seen is by having this message be the very last thing printed. Otherwise it gets 
 lost in the noise.

Reply:
decided to make it better and add support for winHTTP. I was hoping to have it since some time ago already.